commands/Fun/dogfacts.js

```diff
@@ -0,0 +1,124 @@
+  "In 1957, Laika became the first living being in space via an earth satellite and JFK’s terrier, Charlie, fathered 4 puppies with Laika’s daughter.",
+  "The most popular dog breed in Canada, U.S., and Great Britain is the Labrador retriever.",
+  "Although it was once illegal to keep dogs as pets in Iceland's capital city, the laws have been relaxed.",
+  "The Basenji is the only barkless dog.",
+  "Dogs are capable of understanding up to 250 words and gestures, can count up to five and can perform simple mathematical calculations. The average dog is as intelligent as a two-year-old child.",
+  "Basset Hounds cannot swim.",
+  "The phrase \"raining cats and dogs\" originated in seventeenth-century England. During heavy rainstorms, many homeless animals would drown and float down the streets, giving the appearance that it had actually rained cats and dogs.",
+  "The Labrador is so popular, in 2006 there were approximately 3-5 times more Labs as there were German Shepherds or Golden Retrievers.",
+  "Male dogs will raise their legs while urinating to aim higher on a tree or lamppost because they want to leave a message that they are tall and intimidating. Some wild dogs in Africa try to run up tree trunks while they are urinating to appear to be very large.",
+  "Even with their poor eyesight, dogs can still see better at night than humans do.",
+  "An elderly woman was saved by her 12-pound Yorkshire Terrier, who fought off an 80- pound Akita, and survived with only 9 stitches.",
+  "Unlike cats, dogs are not exclusively carnivores: they can adapt to different foods, and a percentage of their diets can be fruits and vegetables.",
+  "Female wolves have been known to travel great distances to regurgitate full meals for their hungry pups.",
+  "There are hundreds of breeds of dogs.",
+  "In Egypt, a person bitten by a rabid dog was encouraged to eat the roasted liver of a dog infected with rabies to avoid contracting the disease. The tooth of a dog infected with rabies would also be put in a band tied to the arm of the person bitten. The menstrual blood of a female dog was used for hair removal, while dog genitals were used for preventing the whitening of hair.",
+  "Dogs can get jealous when their humans display affection toward someone or something else.",
+  "Weird dog laws include allowing police offers in Palding, Ohio, to bite a dog to quiet it. In Ventura County, California, cats and dogs are not allowed to have sex without a permit.",
+  "The largest dog was an English Mastiff who weighed 343 pounds.",
+];
+exports.run = (client, msg) => {
+  msg.channel.send(facts[Math.floor(Math.random() * facts.length)]);
+};
+
+exports.help = {
```

Comment:
To bring consistency with the rest of pieces, please put `exports.conf` above `exports.help`